spec/search_builder/sufia/my_shares_search_builder_spec.rb

```diff
@@ -22,4 +24,28 @@
                            "{!terms f=has_model_ssim}GenericWork,Collection",
                            "-depositor"]
   end
+
+  describe "mediated deposit" do
+    before do
+      allow(Flipflop).to receive(:enable_mediated_deposit?).and_return(mediation_enabled)
+    end
+
+    context "with mediated deposit enabled" do
+      let(:user_query) { nil }
```

Comment:
Since this `let()` statement has the same value in both of the new context blocks, should it sit above them, beneath the new `before` block?